.github/workflows/CI.yml

```diff
@@ -0,0 +1,38 @@
+name: CI
+
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+
+jobs:
+  npm_vulnerabilities_check:
+    name: NPM dependency vulnerabilities check
+    uses: vazco/open-standards/.github/workflows/check_dependency_vulnerabilities.yml@master
+
+  CI:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        meteor-version: ['2.5.6', '2.6.1', '?'] # ? - latest meteor version
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Npm dependency duplication check
+        uses: vazco/open-standards/actions/npm-dependency-duplications/@action-test
+
+      - name: Install Meteor
+        env:
+          METEOR_VERSION: ${{ matrix.meteor-version }}
+        run: curl https://install.meteor.com/?release=$METEOR_VERSION | /bin/sh
```

Comment:
Would probably be worth caching both meteor and npm install dependencies - doesn't have to be within this PR though.